modulles/news.modules.js

```diff
@@ -28,3 +28,16 @@ exports.writeEndpoints = (endpoints) => {
   const endpointsFile = "endpoints.json";
   return fs.writeFile(endpointsFile, JSON.stringify(endpoints, 2));
 };
+
+exports.countComments = () => {
```

Comment:
I would potentially name this something more semantic such as `getArticlesWithCommentCounts` as count comments implies it is counting something